domains/email.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ukrioo",
+        "email": "u8k50850@gmail.com"
+    },
+    "record": {
+        "URL": "https://ukrioo.github.io/"
```

Comment:
note: we are having issues with url record so if its not working as expected contact on discord